pylot/config.py

```diff
@@ -8,15 +8,24 @@
 flags.DEFINE_string('log_file_name', None, 'Name of the log file')
 flags.DEFINE_string('csv_log_file_name', None,
                     'csv file into which to log runtime stats')
-flags.DEFINE_bool('ground_agent_operator', True,
-                  'True to use the ground truth controller')
+
 
 # Sensors to enable.
 flags.DEFINE_bool('lidar', False, 'True to enable the lidar sensor')
 flags.DEFINE_bool('top_down_segmentation', False,
                   'True for enable the top down segmentation camera')
 
-# Modules to enable.
+# Planning modules to enable.
+flags.DEFINE_bool('waypoint_planning_operator', False,
```

Comment:
Similar comments as above. In what circumstances do we need to use the WaypointPlanningOperator over PlanningOperator?

Reply:
See: https://github.com/erdos-project/pylot/pull/37#discussion_r349956067